examples/arduino/LVGL_Arduino/LVGL_Arduino.ino

```diff
@@ -34,7 +34,7 @@ void my_print( lv_log_level_t level, const char * buf )
 #endif
 
 /* LVGL calls it when a rendered image needs to copied to the display*/
-void my_disp_flush( lv_display_t *disp, const lv_area_t *area, uint8_t * px_map)
+void my_disp_flush( lv_display_t *disp, const lv_area_t *area, uint8_t * px_map, uint32_t px_map_size)
```

Comment:
It is not a blocker comment, but I have seeing some PR that break the API and parameter change, would not be good to consider (maybe in a longer term) to pack parameters passed from callbacks or for the API inside of structs? 

It would be easier to maintain and would not break the API itself.